src/gen/pdb2pqr.py

```diff
@@ -5,6 +5,14 @@
 #from beaupy.spinners import *
 from rich.console import Console
 
+def get_file_path():
+    if confirm("Do you have the necessary file downlowded on your device?"):
+        file_path = prompt('Enter the path to your file', target_type=str)      
+    else:
+        pass #yet to do
```

Comment:
If someone already is choosing for self force field, no need to confirm. Directly input and use it, check for existence pls.

Reply:
Also choose better names like instead of file_path, choose ff_path. Makes more sense.

Reply:
Okay, but I don't know how to check for existence

Reply:
> If someone already is choosing for self force field, no need to confirm. Directly input and use it, check for existence pls.

I thought of giving users another option to upload which is why I used the confirm, but ig that isn't necessary

Reply:
> Also choose better names like instead of file_path, choose ff_path. Makes more sense.

Yeah, I genuinely have no clue why that's a better name

Reply:
ff = force field hence ff_path.

Reply:
```py
ff_path = "/path/to/your/file"
if os.path.exists(file_path):
    # your code
    pass
else:
    pass
```

Reply:
done.